### ISSUE TITLE
highlights div at bottom of document needs to be hidden from screen reader (ARIA attribute missing)

### ISSUE THREAD

**Issue by danielweck** [edrlab/thorium-reader#2858]

**Comment by danielweck** [comment]
Fixed.
